certgen/certProcessor/src/test/java/org/incredible/certProcessor/views/PdfConverterTest.java

```diff
@@ -0,0 +1,54 @@
+package org.incredible.certProcessor.views;
+
+import com.itextpdf.html2pdf.HtmlConverter;
+import com.itextpdf.licensekey.LicenseKey;
+import org.incredible.certProcessor.JsonKey;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.File;
+import java.io.InputStream;
+
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({PdfConverter.class, HtmlConverter.class, LicenseKey.class})
+@PowerMockIgnore("javax.management.*")
+public class PdfConverterTest {
+
+  File htmlFile = null;
+
+  @Before
+  public void setUp() {
+    htmlFile = new File("src/test/resources/ValidTemplate.html");
+  }
+
+  @Test
+  public void convertorTest() throws Exception {
```

Comment:
Test case need to have assert condition

Reply:
We are testing a void function, so now added verify functionality in order to check whether it is actually executed or not.